scripts/scripts.js

```diff
@@ -120,12 +120,29 @@ export function formatDate(date) {
   });
 }
 
+/**
+ * Builds an article sidebar and appends it to main in a new section.
+ * @param main
+ */
+async function buildArticleSidebar(main) {
+  if (main.querySelector('.sidebar') || !main.firstChild) {
+    // already got a sidebar, !main.firstChild avoids recursive rendering for empty `main` blocks
+    return;
+  }
+  const locInfo = getLocaleInfo();
+  const sidebarBlock = buildBlock('fragment', [
+    [a({ href: `${locInfo.placeholdersPrefix}/fragments/sidebar-fragment` }, 'Sidebar')],
+  ]);
+
+  main.append(div(sidebarBlock));
```

Comment:
do we need to call `detectSidebar(main)` here to have the correct styling in place?